src/ansys/dpf/post/fluid_simulation.py

```diff
@@ -223,16 +236,43 @@ def _get_result_workflow(
         qualifiers: Union[dict, None] = None,
     ) -> (dpf.Workflow, Union[str, list[str], None], str):
         """Generate (without evaluating) the Workflow to extract results."""
```

Comment:
I took some code out of the old _build_result_workflow. The assumption here is that cyclic expansion is not supported for fluid simulations. I cannot use the standard initial_result_workflow because the operator needs to be exposed and is used on line 311.